app/core/Router.coffee

```diff
@@ -226,6 +227,8 @@ module.exports = class CocoRouter extends Backbone.Router
       return @navigate('/students', {trigger: true, replace: true}) if me.isStudent() and not me.isAdmin()
       @routeDirectly('teachers/ConvertToTeacherAccountView', [])
 
+    'school-administrator(/*subpath)': go('core/SingletonAppVueComponentView')
```

Comment:
Can we use vue-router for any new vue pages going forward, or is this specific to admin dashboard? If its generic, then we should add some general how-to documentation, and any specific things to keep in mind since we also have a backbone router.

Reply:
Added a task to track this